utils/create-cluster/create-cluster

```diff
@@ -65,7 +65,7 @@ then
         sed -i.bak "s|port.*|port ${PORT}|g" ${conf_file} && rm ${conf_file}.bak
         sed -i.bak "s|dir.*|dir "node_${PORT}"|g" ${conf_file} && rm ${conf_file}.bak
         $BIN_PATH/kvrocks -c ${conf_file}
-        sleep 1
+        sleep 2 # wait enough time
```

Comment:
Can you share a bit more about how you evaluate this parameter? That is, why choose 2?

Reply:
Wait 1 second  will `Connection refused` on my test machine, 2 seconds is fine

Reply:
```
./create-cluster start
Starting 30001
OK
OK
Starting 30002
OK
OK
Starting 30003
OK
OK
Starting 30004
Could not connect to Redis at 127.0.0.1:30004: Connection refused
Could not connect to Redis at 127.0.0.1:30004: Connection refused
Starting 30005
Could not connect to Redis at 127.0.0.1:30005: Connection refused
Could not connect to Redis at 127.0.0.1:30005: Connection refused
Starting 30006
OK
OK
```

Reply:
I think we can detect if the port is serving like the below:

```shell
until nc -z 127.0.0.1 3004; do
  echo "instance is not ready"
  sleep 1
done
```

Reply:
> 
a good idea